lib/human_player.rb

```diff
@@ -0,0 +1,40 @@
+class HumanPlayer
+  attr_accessor :mark
+
+  def initialize(mark, board, display)
+    @board = board
+    @display = display
+    @mark = mark
+  end
+
+  def move
+    @display.make_move_message(@mark)
+    index = @display.retrieve_user_input.to_i - 1
+    @board.valid_move?(index) ? index : move
+  end
+
+  def create_mark(other_mark = nil)
```

Comment:
This method should probably be located in `PlayerCreator`. Like, determine which mark a player needs and then initialize a new player with that mark. Otherwise, how will you specify a mark for a computer player?